csm/plugins/cortx/provisioner.py

```diff
@@ -251,23 +252,27 @@ async def get_network_configuration(self):
         :returns: an instance of NetworkConfiguirationResponse
         :raises: a CsmInternalError in case of query failure
         """
-        if not self.provisioner:
-            raise NetworkConfigFetchError(const.PROVISIONER_PACKAGE_NOT_INIT)
-
-        def _command_handler():
-            try:
-                response = self.provisioner.get_params(self.PRVSNR_NETWORK_PARAM_VIP, self.PRVSNR_NETWORK_PARAM_CIP)
-                # The IPs are same for each node, so we can take any of them
-                for node, params in response.items():
-                    return NetworkConfiguirationResponse(
-                        mgmt_vip=params[self.PRVSNR_NETWORK_PARAM_VIP],
-                        cluster_ip=params[self.PRVSNR_NETWORK_PARAM_CIP]
-                    )
-            except Exception as error:
-                Log.error(f"Provisioner api error : {error}")
-                raise NetworkConfigFetchError(f"Network configuration fetching failed: {error}")
-
-        return await self._await_nonasync(_command_handler)
+        if not self.network_config:
+            Log.debug("Netword config is not present in in-memory.")
```

Comment:
Please correct spelling of Networkd to network

Reply:
Done